data_validation/consts.py

```diff
@@ -119,10 +119,12 @@
 VALIDATION_STATUS = "validation_status"
 VALIDATION_STATUS_SUCCESS = "success"
 VALIDATION_STATUS_FAIL = "fail"
+VALIDATION_STATUS_WARNING = "warning"
 # TODO: update if we start to support other statuses
 VALIDATION_STATUSES = [
     VALIDATION_STATUS_SUCCESS,
     VALIDATION_STATUS_FAIL,
+    VALIDATION_STATUS_WARNING,
```

Comment:
We shouldn't add another status as 'warning', but rather throw a warning with logging.warning if a data type isn't an exact match but still fits within the target type (i.e. decimal(1,1) --> decimal(38,9) )
In this case, it should still be marked as a validation success.